Makcym/golang_learning_project
# Setup&Docker to beego-api

AC:
* Dockerize beegoo-api

Steps to test:
* Run `make build`
* Open http://localhost:8881/v1/user to check
